src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -439,6 +441,7 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configOnlySyncPact = False
     , _configSyncPactChains = Nothing
     , _configBackup = defaultBackupConfig
+    , _configModuleCacheLimit = 10 * 1024^(2::Int) -- 10MiB (per chain)
```

Comment:
Seems like a small default. I guess we will revisit this later?

Reply:
Yes, it's a bit too small, in particular for catchup and replay, based on a number of experiments that I performed. In the most recent versions I increased it to 60MiB per chain, which seems fine for prod and also gives decent results for replay. Although for replay it's probably better to configure something larger -- depending on available memory.